### PR TITLE
Nuka cola speed buff nerf

### PR DESCRIPTION
**Pull request by Varghh**
## What Does This PR Do
Nuka cola speed nerfed, it's now a budged version of meth speed.

## Why It's Good For The Game
Nuka Cola speed is too easy to get and speed is king in this game, this PR change how much you speed up when you drink it.
Normal Speed 2.5
Nuka Speed 2
Meth Speed 1.5

## Changelog
:cl:
tweak: Nuka Cola speed nerfed from 1 to 0.5.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drink_cold.dm [hunk 1]
```diff
@@ -74,6 +74,7 @@
 	M.AdjustDizzy(5)
 	M.SetDrowsy(0)
 	M.status_flags |= GOTTAGOFAST
+	M.apply_effect(2, IRRADIATE, negate_armor = 1)
```
Comment (moxian) [inline code/modules/reagents/chemistry/reagents/drink_cold.dm:77]:
```suggestion
	M.apply_effect(2, IRRADIATE, negate_armor = TRUE)
```

**Comment by procdrone** [comment]
Good idea, though radiation can be easily sidestepped just taking an additional pottasium iode pill, which isn't very hard to obtain.

... maybe making nuka cola also being an addictive chemical?

**Comment by Mitchs98** [comment]
God yes.

**Comment by Spacemanspark** [comment]
I'm going to laugh when we finally add nucleations and they get speed AND a heal off of this lmfao

**Comment by Kiyahitayika** [comment]
I support this and have a few questions:
1.) What do you think about adding a warning message for players? "You don't feel so great after drinking that . . ."
2.) Can Radiation damage cause mutation or is that separate?
3.) How much radiation is it per sip? Two? (asking for Medbay)

@procdrone Potassium Iodine bottles are stocked in the NanoMed Vendors in Medbay and, yet . . . Baldbay doesn't know how to treat Radiation damage very often, soooo . . . x3

**Comment by Varghh** [comment]
> 1.) What do you think about adding a warning message for players? "You don't feel so great after drinking that . . ."

Taste is already _"radioactive cola"_, the drink flash green lights, has a radioactive logo and it's literally a Fallout drink... I think there are enough warnings.. :p

> 2.) Can Radiation damage cause mutation or is that separate?

Yeah, depends on how much radiation you have, a bit of rad should not be a problem, a lot of rad will fuck you up.

> 3.) How much radiation is it per sip? Two? (asking for Medbay)

It should be 2 per cycle.

**Review by SteelSlayer — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]

**Review by Kyep — APPROVED** [review]

**Comment by Fox-McCloud** [comment]
Why don't we try nerfing how much it makes you faster.

**Comment by Varghh** [comment]
> Why don't we try nerfing how much it makes you faster.

On top of the radiation?

**Comment by hal9000PR** [comment]
If it gave less speed then it wouldn't have the gottagofast flag and it would stack with meth again. Personally I think giving it radiation is perfectly fine from a balance standpoint because it means that orm access doesn't mean free speed.

**Review by variableundefined — CHANGES_REQUESTED** [review]
@Varghh We would prefer nerfing how fast it goes **instead** of giving it radiation for two reasons: 
- It is easy to counteract radiation 
- Going to medbay for a day to day drink is weird

**Comment by Darkmight9** [comment]
Going off of Ansari above, why does it give a speed boost in the first place anyway? None of the other uranium drinks do anything extra other than look neat and get you drunk.

**Comment by Varghh** [comment]
Per request, Nuka cola now only speeds you up by 0.5 instead of 1, radiation removed
@variableundefined

**Pull request merged**